src/app/sidebar-right/sidebar-right.component.ts

```diff
@@ -1,17 +1,24 @@
 import { Component, OnInit } from '@angular/core';
+import { MapService } from '../shared/services/map.service';
 
 @Component({
   selector: 'app-sidebar-right',
   templateUrl: './sidebar-right.component.html',
   styleUrls: ['./sidebar-right.component.scss']
 })
 export class SidebarRightComponent implements OnInit {
-
+	
 	popout = '';
 
-  	constructor() { }
+	currentClick!: object;
+
+  	constructor(private _mapService: MapService) { }
 
-  	ngOnInit(): void {
+  	ngOnInit(): void {  
+		// Getting current click point from map service to communicate to report component
+		this._mapService.clickPoint.subscribe((point: object) => {
+			this.currentClick = point; 
+		});
```

Comment:
Instead of the sidebar-right getting this, the report component can just subscribe to the click point, then you won't need the `@Input` in the report component.

Also worth noting that in the future, I think we'll create a subject/observable with the outputs for the report that we'll update using the workflows, rather than having the report subscribe to the click points, etc.